tests/integration/components/preprint-footer-branded-test.js

```diff
@@ -1,51 +1,20 @@
+import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
+import sanitizer from 'preprint-service/sanitizers/footer-links';
 
 moduleForComponent('preprint-footer-branded', 'Integration | Component | preprint footer branded', {
-  integration: true
-});
-
-test('shows Twitter', function(assert) {
-    this.set('model', {
-        socialTwitter: 'vincestaples'
-    });
-
-    this.render(hbs`{{preprint-footer-branded model=model}}`);
-
-    assert.equal(this.$('.fa-twitter').length, 1);
-    assert.equal(this.$('.fa-facebook').length, 0);
-    assert.equal(this.$('.fa-instagram').length, 0);
-});
+  integration: true,
 
-test('shows Facebook', function(assert) {
-    this.set('model', {
-        socialFacebook: 'theWeatherStn'
-    });
+  setup() {
+    this.container.registry.register('sanitizer:footer-links', Ember.Object.extend(sanitizer));
+  }
 
-    this.render(hbs`{{preprint-footer-branded model=model}}`);
-
-    assert.equal(this.$('.fa-twitter').length, 0);
-    assert.equal(this.$('.fa-facebook').length, 1);
-    assert.equal(this.$('.fa-instagram').length, 0);
-});
-
-test('shows Instagram', function(assert) {
-    this.set('model', {
-        socialInstagram: 'welpherewego'
-    });
-
-    this.render(hbs`{{preprint-footer-branded model=model}}`);
-
-    assert.equal(this.$('.fa-twitter').length, 0);
-    assert.equal(this.$('.fa-facebook').length, 0);
-    assert.equal(this.$('.fa-instagram').length, 1);
 });
 
 test('shows all social', function(assert) {
     this.set('model', {
-        socialTwitter: 'vincestaples',
-        socialFacebook: 'theWeatherStn',
-        socialInstagram: 'welpherewego'
+        footerLinks: '<a class="fa fa-2x fa-twitter"></a><a class="fa fa-2x fa-instagram"></a><a class="fa fa-2x fa-facebook"></a>'
     });
 
```

Comment:
Could you add sanitizer tests? Don't let it hold up the product demo, though. Just make sure all the tags you want go through, and one or two tags you don't want are sanitized.